addons/binding/org.openhab.binding.lutron/ESH-INF/thing/thing-types.xml

```diff
@@ -167,6 +167,154 @@
         </config-description>
     </thing-type>
 
+    <bridge-type id="prgbridge">
+        <label>Lutron GRX-PRG or GRX-CI-PRG Bridge</label>
+        <description>Ethernet access point to Lutron Grafik Eye 3x/4x Systems</description>
+
+        <channels>
+            <channel id="buttonpress" typeId="buttonpress"/>
```

Comment:
The PRG interfaces have the ability to report what buttons have been pressed throughout the system (front panel, any keypad, etc).  That channel will report (readonly) which button was pressed - allowing a rule to trigger off something based on that (such as if you hit the third button on the fifth keypad).  

This gives you more granularity on button presses than scene reporting does (since multiple buttons can be linked to a single scene).

I'll add some more documentation to the README.MD to clarify - sound ok?